WTWSMS/history/titles/k_nicene.txt

```diff
@@ -0,0 +1,14 @@
+37.1.1={
+	
+	liege="e_byzantium"
+	
+
+	active = yes
+
+	
+
+	government = theocracy_government
+	
+	succession = succ_open_elective
+
+}
```

Comment:
No, you are mistaken. The bug removed the entire concept of a religious head AND broke the UI.

In the graphics, there's one big picture of the main religious head that Pentarchy defaults to, and then there are the four other Pentarchs. The bug I mentioned breaks those pictures and makes one float outside of the window, as well as messing with the titles, names, and alignment of the text.

Trust me, it's a serious issue that sometimes lead to the entire religious head mechanics not registring. With this bug, I had "there is and cannot exist a religious head" for MIAPHYSITES.